CMake/fletch-tarballs.cmake

```diff
@@ -121,24 +121,30 @@ if ( NOT fletch_ENABLE_OpenCV OR OpenCV_SELECT_VERSION VERSION_LESS 3.0.0 )
   unset(fletch_ENABLE_OpenCV_contrib CACHE)
 endif()
 
-if (OpenCV_SELECT_VERSION VERSION_EQUAL 3.1.0)
-  set(OpenCV_version "3.1.0")
+if (fletch_ENABLE_OpenCV OR fletch_ENABLE_ALL_PACKAGES)
+  set(OpenCV_version ${OpenCV_SELECT_VERSION})
   set(OpenCV_url "http://github.com/Itseez/opencv/archive/${OpenCV_version}.zip")
+  set(OpenCV_dlname "opencv-${OpenCV_version}.zip")
+  if (fletch_ENABLE_OpenCV_contrib OR fletch_ENABLE_ALL_PACKAGES)
+    set(OpenCV_contrib_version "${OpenCV_version}")
+    set(OpenCV_contrib_url "http://github.com/Itseez/opencv_contrib/archive/${OpenCV_contrib_version}.zip")
+    set(OpenCV_contrib_dlname "opencv-contrib-${OpenCV_version}.zip")
+  endif()
+endif()
+
+# Paired contrib repo information
+if (OpenCV_version VERSION_EQUAL 3.3.0)
+  set(OpenCV_md5 "cc586ebe960a7cdd87100e89088abc06")
+  set(OpenCV_contrib_md5 "2dd6dc53d49a09dd8538e63a55edc87a")
+elseif (OpenCV_version VERSION_EQUAL 3.1.0)
   set(OpenCV_md5 "6082ee2124d4066581a7386972bfd52a")
-  # Paired contrib repo information
-  set(OpenCV_contrib_version "${OpenCV_version}")
-  set(OpenCV_contrib_url "http://github.com/Itseez/opencv_contrib/archive/${OpenCV_contrib_version}.zip")
   set(OpenCV_contrib_md5 "0d0bfeabe539542791b465ec1c7c90e6")
-  set(OpenCV_contrib_dlname "opencv-contrib-${OpenCV_version}.zip")
-elseif (OpenCV_SELECT_VERSION VERSION_EQUAL 2.4.11)
-  set(OpenCV_version "2.4.13")
-  set(OpenCV_url "http://github.com/Itseez/opencv/archive/${OpenCV_version}.zip")
+elseif (OpenCV_version VERSION_EQUAL 2.4.13)
```

Comment:
Hmm, I didn't think I bumped the minor version of `2.4`. 

Looking at only the context of the diff isn't this fixing a typo? We were always using 2.4.13, but the `OpenCV_SELECT_VERSION` was just switching on the value `2.4.11` for whatever reason.